### PR TITLE
Normalize with Uno prefix and _v0 on task names

### PR DESCRIPTION
**Pull request by dansiegel**
GitHub Issue (If applicable): #

- n/a

## PR Type

What kind of change does this PR introduce?

- Refactoring

### CODE REVIEW

File: src/.nuspec/Uno.Resizetizer.targets [hunk 1]
```diff
@@ -55,7 +55,7 @@
 	<PropertyGroup>
 		<CleanDependsOn>
 			$(CleanDependsOn);
-			_CleanResizetizer;
```
Comment (jeromelaban) [inline src/.nuspec/Uno.Resizetizer.targets:58]: Why is this needed? Is it to avoid clashes with the original resizetizer? I'd think that the uno prefix should be the only change needed, as the `_v0` is only to disambiguate the class and assembly names at runtime.

Comment (dansiegel) [reply src/.nuspec/Uno.Resizetizer.targets:58]: Yeah thought this was weird... that was miscommunicated as somehow requirement.

**Review by jeromelaban — APPROVED** [review]

**Comment by dansiegel** [comment]
@Mergifyio backport release/stable/1.1
